ote/src/clj/ote/services/transport.clj

```diff
@@ -566,6 +570,21 @@
           (authorization/with-transport-operator-check db user operator-id
                                                        #(delete-associated-operator db service-id operator-id))))
 
+      (GET "/transport-operator/:operator-id/users"
+           {{:keys [operator-id]}
+            :params
+            user :user}
+        (let [op-id (Long/parseLong operator-id)]
+          (authorization/with-transport-operator-check db user op-id
+                                                       #(transport-operator-users db op-id))))
+
+      (POST "/transport-operator/:operator-id/users"
```

Comment:
Should we check that requester have access to given operator

Reply:
Definetly, but I tought it would be done in the task where the backend service is built. Since this is just a placeholder.